README.md

```diff
@@ -110,6 +119,7 @@ function App() {
 | **perLine** | `9` | | The number of emojis to show per line |
 | **previewEmoji** | `point_up` | | The id of the emoji to use for the preview when not hovering any emoji. `point_up` when preview position is bottom and `point_down` when preview position is top. |
 | **previewPosition** | `bottom` | `top`, `bottom`, `none` | The position of the preview |
+| **rtl** | `false` | | Whether the Right to Left style or not |
```

Comment:
I’m not sure if we really need a prop for this 🤔. This should be controlled by the `i18n` data. Chances are there is not much use case of manually providing `rtl` without also providing a language that is rtl.